packages/ui/components/form/checkbox/Checkbox.tsx

```diff
@@ -53,13 +55,11 @@ const CheckboxField = forwardRef<HTMLInputElement, Props>(
                     type="checkbox"
                     disabled={disabled}
                     className={classNames(
-                      "text-primary-600 focus:ring-primary-500 border-default bg-default h-4 w-4 rounded ltr:mr-2 rtl:ml-2",
-                      !error && disabled
-                        ? "bg-gray-300 checked:bg-gray-300"
-                        : "hover:bg-subtle checked:bg-gray-800",
+                      "border-default h-4 w-4 rounded ltr:mr-2 rtl:ml-2",
                       error && "border-red-800 checked:bg-red-800 hover:bg-red-400",
                       rest.className
                     )}
+                    style={{ color: `${user?.brandColor}` }}
```

Comment:
are we sure we want brandColor here?

Reply:
Or is the Checkbox not supposed to be in brandColor?

Reply:
> Do you mean we should move this brandColor to className?

we should move it to the classNames i have mentioned below :)